scripts/zkai_proof_pressure_main_evidence_gate.py

```
@@ -0,0 +1,411 @@
+        "all_mutations_rejected": True,
+    }
+    validate_payload(payload)
+    return payload
+
+
+def validate_payload(payload: dict[str, Any]) -> None:
+    if payload.get("schema") != SCHEMA or payload.get("decision") != DECISION:
+        raise ProofPressureMainEvidenceError("payload identity drift")
+    rows = payload.get("rows")
+    if not isinstance(rows, list) or len(rows) != 5:
+        raise ProofPressureMainEvidenceError("evidence row smuggling")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)